acond/src/utils.rs

```diff
@@ -41,9 +41,9 @@ pub const ERR_RPC_REJECT_BLOB: &str = "Reject blob as no existing manifests requ
 pub const ERR_RPC_INVALID_IMAGE_ID: &str = "Invalid image id.";
 pub const ERR_RPC_INVALID_CONTAINER_ID: &str = "Invalid container id.";
 pub const ERR_RPC_CONTAINER_EXITED: &str = "Container has exited.";
-pub const ERR_RPC_CONTAINER_NOT_ALLOW_KILL: &str = "Container is not allowed to be killed.";
-pub const ERR_RPC_CONTAINER_KILL_TIMEOUT: &str = "Container can't be killed in {} seconds.";
+pub const ERR_RPC_CONTAINER_RESTART_TIMEOUT: &str = "Container can't be restarted in {} seconds.";
 pub const ERR_RPC_CONTAINER_NOT_ALLOW_RESTART: &str = "Container is not allowed to be restarted.";
+pub const ERR_RPC_CONTAINER_NOT_ALLOW_KILL: &str = "Container is not allowed to be killed.";
```

Comment:
`"Signal not allowed"`?

Reply:
So as discussed, all these error messages should be refactored as the same style - like "Timeout restarting container" and "Signal not allowed"?

Reply:
Agree